src/AppHost/Program.cs

```diff
@@ -0,0 +1,5 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddProject<Projects.Web>("web");
+
+builder.Build().Run();
```

Comment:
Should we add a SQL Server component here? The template uses a LocalDb by default.

Reply:
Thanks for the suggestion. I've updated it to include SQL Server integration.

Reply:
Hi, I'm trying to setup a new WebAPI solution using "--use-aspire" but it's not working, and says -is not a valid option.
Could you please suggest the right commend to create a new Aspire solution from template?

Reply:
> Hi, I'm trying to setup a new WebAPI solution using "--use-aspire" but it's not working, and says -is not a valid option. Could you please suggest the right commend to create a new Aspire solution from template?

This PR is not merged and not released yet, so you'd have to build and install the templates from this branch to use the flag.

Reply:
I tried below and it gave me same error. Am I missing something?

![Uploading image.png…]()

Reply:
pramodjaiswal@192 CleanArchitecture % dotnet new ca-sln --use-aspire -cf None -o OrchestrationEngine
Error: Invalid option(s):
--use-aspire
   '--use-aspire' is not a valid option

Reply:
Sorry, that will not work until this PR is merged and a NuGet package is published. I'll try to get it done today.